packages/cli/src/util/dev/server.ts

```diff
@@ -2185,13 +2186,7 @@ function proxyPass(
         `Failed to complete request to ${req.url}: ${error}`
       );
       if (!res.headersSent) {
-        devServer.sendError(
-          req,
-          res,
-          requestId,
-          'NO_RESPONSE_FROM_FUNCTION',
-          502
-        );
+        devServer.sendError(req, res, requestId, 'FUNCTION_INVOCATION_FAILED');
```

Comment:
`FUNCTION_INVOCATION_FAILED` is a better message here and it matches production. This should be a 500 (the default for `sendError`), not 502.